installers/olm/description.upstream.md

```diff
@@ -49,20 +49,12 @@ export PGO_OPERATOR_NAMESPACE=pgo
 kubectl create namespace "$PGO_OPERATOR_NAMESPACE"
 ```
 
-### Secrets
+### Secrets (optional)
 
-Configure pgBackRest for your environment. If you do not plan to use AWS S3 to store backups, you can omit
-the `aws-s3` keys below.
+If you plan to use AWS S3 to store backups, you can configure your environment to automatically provide you AWS S3 credentials to all newly created PostgreSQL cluster:
```

Comment:
Looks like should be "...provide _your_ AWS S3 credentials...

And then also "clusters" at the end, e.g. "...newly created PostgreSQL clusters"